Add debugging instructions (#179)
statechannels/go-nitro: contributing.md

```shell
go mod tidy
```

> # Debugging Tests
> 
> VS code is used to debug tests. To start a debugging session in VS code:
> 
> - Open the test file.
> - Open the `Run and Debug` section.
> - Run the `Debug Test` configuration.
> 
> The [go extension](https://marketplace.visualstudio.com/items?itemName=golang.Go) also supports debugging and can be used to start debugging session right from a test function.

Reviewer: Nit: I think the extension is required to do any kind of debugging:-
```suggestion
# Debugging Tests

VS code is used to debug tests. To start a debugging session in VS code:

- Ensure you have the [go extension](https://marketplace.visualstudio.com/items?itemName=golang.Go) installed
- Open the test file.
- Open the `Run and Debug` section.
- Run the `Debug Test` configuration.

With the extension it is also possible to start a debugging session right from a test function.
```
